server/utils/sendBill.js

```diff
@@ -0,0 +1,71 @@
+const nodemailer = require('nodemailer');
+
+const sendTheBill = async (
+  to,
+  subject,
+  { total, hourPrice, hourNumber, description, client, provider },
```

Comment:
more clear to pass the object of email content and destructure the value , like this :
```js
const sendTheBill = async (to, subject, content, next) => {
  try {
    const {
      total,
      hourPrice,
      hourNumber,
      description,
      client,
      provider,
    } = content;
```